dune_client/viz/graphs.py

```diff
@@ -0,0 +1,83 @@
+"""
+Functions you can call to make different graphs
+"""
+
+import plotly.graph_objects as go
+import colorlover as cl
+import pandas as pd
+
+# function to create Sankey diagram
+def create_sankey(
+    query_result: pd.DataFrame,
+    predefined_colors: dict,
+    columns: dict,
+    viz_config: dict,
```

Comment:
the test suggests that these could be 
```suggestion
    predefined_colors: dict[str, str],
    columns: dict[str, str],
    viz_config: dict[str, int | float],
```

Although it also kinda looks like columns could be `list[str]`. 

Note that earlier python versions might require `Union[int, float]` (if you decide to go with this suggestion).